packages/mux-player/test/player.test.js

```diff
@@ -197,6 +197,24 @@ describe('<mux-player>', () => {
     assert.equal(muxVideo.preload, 'auto', `has preload enabled`);
   });
 
+  it('preload behaves like expected', async function () {
+    const player = await fixture(`<mux-player
+    ></mux-player>`);
+    const muxVideo = player.media;
+
+    assert.equal(player.preload, 'metadata', "Chrome's player default preload is metadata");
```

Comment:
wouldn't this fail if we run this in not chrome?

Reply:
yes for sure but so far we don't test other browsers

Reply:
I guess, should we? And if so, maybe better to make this prepped for that?

Maybe testing it's equal to `document.createElement('video').preload`?

Reply:
question(non-blocking): Yeah I'd rather avoid writing tests for a specific browser if possible. For all browsers, it should never `="none"`, correct? Should we test that condition for now?

Reply:
ah yes good idea. it could be `none` theoretically, like mobile safari on 3G maybe does that

Reply:
Mobile safari does preload none, but I think they keep the value as `auto`.